9.4/Dockerfile.rhel7

```diff
@@ -70,6 +72,9 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
 
 VOLUME ["/var/lib/pgsql/data"]
 
+# {APP_DATA} needs to be accessed by postgres user while s2i assembling
+RUN chown -R 26:0 ${APP_DATA} && \
+    chmod -R ug+rwx ${APP_DATA}
```

Comment:
Thinking again, shouldn't we actually use the `fix-permissions` script few lines above while we are fixing the permissions of datadir?